firmware/qemu/src/bin/log.rs

```diff
@@ -501,6 +501,32 @@ fn main() -> ! {
     defmt::info!("{:?}", 340282366920938u128);
     defmt::info!("{:?}", -170141183460469i128);
 
+    {
+        #[rustfmt::skip]
+        #[allow(dead_code)]
+        #[derive(Format)]
+        enum EnumLarge {
+            A000, A001, A002, A003, A004, A005, A006, A007, A008, A009, A010, A011, A012, A013, A014, A015, A016, A017,
+            A018, A019, A020, A021, A022, A023, A024, A025, A026, A027, A028, A029, A030, A031, A032, A033, A034, A035,
+            A036, A037, A038, A039, A040, A041, A042, A043, A044, A045, A046, A047, A048, A049, A050, A051, A052, A053,
+            A054, A055, A056, A057, A058, A059, A060, A061, A062, A063, A064, A065, A066, A067, A068, A069, A070, A071,
+            A072, A073, A074, A075, A076, A077, A078, A079, A080, A081, A082, A083, A084, A085, A086, A087, A088, A089,
+            A090, A091, A092, A093, A094, A095, A096, A097, A098, A099, A100, A101, A102, A103, A104, A105, A106, A107,
+            A108, A109, A110, A111, A112, A113, A114, A115, A116, A117, A118, A119, A120, A121, A122, A123, A124, A125,
+            A126, A127, A128, A129, A130, A131, A132, A133, A134, A135, A136, A137, A138, A139, A140, A141, A142, A143,
+            A144, A145, A146, A147, A148, A149, A150, A151, A152, A153, A154, A155, A156, A157, A158, A159, A160, A161,
+            A162, A163, A164, A165, A166, A167, A168, A169, A170, A171, A172, A173, A174, A175, A176, A177, A178, A179,
+            A180, A181, A182, A183, A184, A185, A186, A187, A188, A189, A190, A191, A192, A193, A194, A195, A196, A197,
+            A198, A199, A200, A201, A202, A203, A204, A205, A206, A207, A208, A209, A210, A211, A212, A213, A214, A215,
+            A216, A217, A218, A219, A220, A221, A222, A223, A224, A225, A226, A227, A228, A229, A230, A231, A232, A233,
+            A234, A235, A236, A237, A238, A239, A240, A241, A242, A243, A244, A245, A246, A247, A248, A249, A250, A251,
+            A252, A253, A254, A255, A256, A257, A258, A259, A260, A261, A262, A263, A264, A265, A266, A267, A268, A269,
+        }
+
+        defmt::info!("Format EnumLarge Variant: {:?}", EnumLarge::A007);
+        defmt::info!("Format EnumLarge Variant: {:?}", EnumLarge::A269);
```

Comment:
this fails locally with 
``` console
0.000101 INFO Format EnumLarge Variant: A007
failed to decode defmt data: [0, 99, 1]
Error: malformed data
```
I think the decoder expects that the *discriminant* is encoded as a single byte but with these changes it should consider how many the variants the enum has (and use 1,2,3,4,etc. bytes as the discriminant size)

Reply:
Whoops interesting. It's failing locally for me as well. I just pushed it to have a failing case to move along and eventually make it pass. 

But that this does not fail in CI is interesting.

Reply:
Well I guess it is because I have not added any `.out` files? :thinking:

Reply:
I think CI is not running because the PR has merge conflicts. The last CI run is from your very first version, which didn't have the qemu test.